### DISCUSSION TITLE
Upgrading PostgREST - how to, and downtime

### DISCUSSION THREAD

**Discussion by zane-atomus** [supabase/supabase#27786] [Questions]
We recently deployed a feature that has more complex ordering/filtering/paging functionality then we previously had. All functions worked in our development Supabase project, however prod is giving the message "failed to parse order". After reviewing what could be causing this, we noticed that dev and QA are running PostgREST v11.1.0, whereas prod is still on 10.2.0. We assume this is the issue (let us know if that is not the case). If this is the issue, we are wondering:

1. How can we update the prod PostgREST version? 

3. What would the expected downtime be for this operation?

We found online that doing a pause/unpause on the project would work, but we are on a paid plan and unable to do that. Would restarting the project work, and if so, is that the recommended method?

**Comment by silentworks** [comment]
The recommended method is to send a [support request](https://supabase.com/dashboard/support/new) to Supabase for them to do the upgrade for you.
